duckduckgo/iOS
# Autofill improvements to onboarding for device auth

Task/Issue URL: https://app.asana.com/0/1201462886803403/1203328956136997/f
Tech Design URL:
CC: @THISISDINOSAUR 

**Description**:
When a user does not have any saved logins, allow them to access the Autofill Logins screen + create / view / edit a new login without requiring authentication. Note that this is subject to outcome of security triage review



**Steps to test this PR**:
1. From a clean install, access the Autofill Logins screen and confirm you are not prompted to authenticate
2. Add a new login, segue back to the Autofill Logins list and confirm you are not prompted to authenticate
3. Tap to view the newly created login and confirm you are not prompted to authenticate
4. Back out of the Autofill Logins screen, then go back in. You should now be prompted to grant biometrics permission and authenticate before you can access the saved logins
5. As a regression test, delete any saved logins and then go to a website and login. Save the login when prompted and tap to view it. Confirm you are prompted to authenticate in order to view it



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
